pkg/cli/admin/migrate/images/imagerefs.go

```diff
@@ -89,10 +89,11 @@ func NewMigrateImageReferenceOptions(streams genericclioptions.IOStreams) *Migra
 func NewCmdMigrateImageReferences(name, fullName string, f kcmdutil.Factory, streams genericclioptions.IOStreams) *cobra.Command {
 	o := NewMigrateImageReferenceOptions(streams)
 	cmd := &cobra.Command{
-		Use:     fmt.Sprintf("%s REGISTRY/NAME=REGISTRY/NAME [...]", name),
-		Short:   "Update embedded Docker image references",
-		Long:    internalMigrateImagesLong,
-		Example: fmt.Sprintf(internalMigrateImagesExample, fullName),
+		Use:        fmt.Sprintf("%s REGISTRY/NAME=REGISTRY/NAME [...]", name),
+		Short:      "Update embedded Docker image references",
+		Long:       internalMigrateImagesLong,
+		Example:    fmt.Sprintf(internalMigrateImagesExample, fullName),
+		Deprecated: "it is not usable in self-managed 4.x cluster",
```

Comment:
This message doesn't make a ton of sense - might be more concise to say "migration of content is managed automatically in OpenShift 4" even if it's not quite true yet, some of these are.

Reply:
I'll open a follow-up to clarify